config/routes.rb

```diff
@@ -15,6 +15,8 @@
 
   patch "/users/:id", to: "users#update", as: "update_user"
 
+  delete "/users/:id", to: "users#destroy", as: "destroy_user"
```

Comment:
No path alias needed. The path here is the same as the path for users#show, so use the user_path instead.